src/machines/authMachine.ts

```diff
@@ -267,7 +335,9 @@ async function logout() {
           console.error('Error revoking token:', e)
         }
 
-        await writeTokenFile('')
+        if (environmentName)
+          await writeEnvironmentConfigurationToken(environmentName, '')
```

Comment:
```suggestion
        if (environmentName) {
          await writeEnvironmentConfigurationToken(environmentName, '')
        }

```

I think this helps with clarity and debug later

Reply:
I believe I'd want the https://eslint.org/docs/latest/rules/curly rule enforced haha, can definitely be its own issue